lib/api/objectDelete.js

```diff
@@ -169,8 +170,14 @@ function objectDelete(authInfo, request, log, cb) {
                     result.versionId : versionIdUtils.encode(
                         result.versionId, config.versionIdEncodingType);
             }
+            const versioningSuspended = bucketMD
+                && bucketMD._versioningConfiguration
+                && bucketMD._versioningConfiguration.status === 'Suspended';
```

Comment:
Cloudserver should support `?.` null chaining I believe
```suggestion
            const versioningSuspended = bucketMD?._versioningConfiguration?.status === 'Suspended';
```